.github/workflows/smoke-test.yml

```
@@ -0,0 +1,23 @@
+name: Smoke test
+
+on:
+  schedule:
+    - cron: '0 0 * * SUN'
+  workflow_dispatch:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 14
+    - run: |
+        npm install
```

i'd prefer to use my node/install action here (which the other actions in the repo also use)
```suggestion
    - uses: ljharb/actions/node/install@main
        name: 'nvm install lts/* && npm install'
        with:
          node-version: 'lts/*'
          skip-ls-check: true
    - run: |
```